manifests/README.md

```diff
@@ -0,0 +1,3 @@
+## Warning 
+Please make sure you are in `claudie` directory before using `kustomize` to deploy Claudie.
+Running `kustomize` in this directory will deploy `testing-framework` along with the other Claudie components.
```

Comment:
I like this, but could it include the information that `testing-framework` isn't a real part of Claudie, and that the user probably doesn't want to deploy it?

Also, what do you think about putting this exact content also into `manifests/testing-framework` ?

Reply:
> I like this, but could it include the information that testing-framework isn't a real part of Claudie, and that the user probably doesn't want to deploy it?
Cool, I'll update it.

> Also, what do you think about putting this exact content also into manifests/testing-framework ?
yeah we can add it imo.